code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,373 @@
+
+|Licenses
+|The usage of open source software licenses composes most of the source codes. At the same time, there might be an oversight regarding the permission and rights to use or repurpose an open source code. In this view, see the listing of licensing issues with remediation solutions through periodic scans.
+
+|Build Integrity
+|Repository misconfigurations occur when multiple owners for repositories exist on platforms like GitHub and GitLab. Through periodic scans, this view highlights the origin of another repository owner and prevents a chance Supply Chain attack.
+
+|VCS Pull Requests
+| After configuring Enforcement, view issues for scans run on all open Pull Requests (PR) for your integrated repositories. On the Prisma Cloud console, you can access the scan result from the console or choose to access the VCS console and view the specific commit within PR for a manual fix.
+
+|CI/CD Runs
+|After configuring Enforcement, view issues for scans performing on runs of your integrated repositories. Using the open PR, you can access the specific run performed on the repository.
+
+|===
+
+You can also find relevant non-conformant scan results using filters or search using keywords or tags (if they are already a part of the code).
+For each misconfiguration on *Projects*, you can either Suppress the issue, Fix it from the Prisma Cloud console, or access the repository in the Version Control System (VCS) to perform a manual fix.
+Currently, this view of *Projects* is accessible on *Code Security > Projects > Enhanced*.
+
+image::proj-1.png[width=800]
+
+=== Resource and Policy Blocks
+
+All issues are viewable as a resource or policy block on *Projects* across code category views.
+
+* *Resource Block*
++
+After periodic scans on resources, Prisma Cloud generates contextualized scanned results of each resource as a resource block. Scan results are issues that are vulnerabilities or code errors found within the resource.
+Each resource block displays only five issues by default. *Show More* in each resource block helps you display more issues within the resource.
+
+* *Policy Block*
++
+After periodic scans, Prisma Cloud generates a policy block. Within it is contextualized scan results with names and lists of all the resources violating the policy.
++
+NOTE: In Vulnerabilities, the grouping of policy blocks is according to the specific CVE severity.
+
+==== Types of Resource and Policy Blocks
+
+Each code category can generate either a resource or a policy block. For understanding the types of blocks corresponding to the code category see the table.
+
+[cols="1,1,1,1,1,1", options="header"]
+|===
+
+|Resource Type/ Code Category
+|IaC Misconfiguration
+|Vulnerabilities
+|Licenses
+|Secrets
+|Build Integrity
+
+|IaC Resource
+|✔️
+|✔️
+|✔️
+|
+|
```

Comment:
@GiladMark is this true? I see only misconfigs in my view, but it's possible I missed CVEs